app/services/candidate_interface/verify_and_mark_references_incomplete.rb

```diff
@@ -0,0 +1,23 @@
+module CandidateInterface
+  class VerifyAndMarkReferencesIncomplete
+    attr_reader :current_application
+
+    def initialize(current_application)
+      @current_application = current_application
+    end
+
+    def call
+      current_application.update!(references_completed: false) if !enough_references? && references_section_is_marked_as_completed?
+    end
+
+  private
+
+    def enough_references?
```

Comment:
We could either delegate here or just not have this aliasing method. To me it's needless code.